apps/monitoring/version-reporter/helm-charts.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: version-reporter-service-helmcharts
+  namespace: monitoring
+  labels:
+    app: version-reporter-service
+spec:
+  schedule: "*/10 * * * *" # interval would be reviewed after testing complete
+  concurrencyPolicy: "Forbid"
+  failedJobsHistoryLimit: 5
+  startingDeadlineSeconds: 300
+  successfulJobsHistoryLimit: 5
+  jobTemplate:
+    spec:
+      backoffLimit: 3
+      template:
+        spec:
+          restartPolicy: Never
+          serviceAccountName: version-reporter-service-sa
+          containers:
+            - name: version-reporter-service-helmcharts
+              image: sdshmctspublic.azurecr.io/hmcts/version-reporter-services:master-helmcharts #{"$imagepolicy": "flux-system:version-reporter"}
```

Comment:
Do we want to update/create tagging policy that matches are standard deployments?

Reply:
@thomast1906 any examples or suggestions you can throw my way on this, thanks

Reply:
like this. https://github.com/hmcts/sds-flux-config/commit/8cebf7aeb58091138474740a5e3dff8ad8fab3dd#diff-c34d4a5c0059b5e971163a1eea92d89b980c18a4f7a9904ce893b38001355a85R17

Reply:
> static image tags are a big No, need to be dynamic preferably with timestamp and commitid like all others.

Would it matter in this case @adusumillipraveen as the apps will not change much, if at all after its complete and does not operate like the cft app would where many changes may make it to master in a period in time

Reply:
It is just a bad practice, we should avoid. this can hadle everything for you https://github.com/hmcts/azure-cftapps-monitoring/blob/master/.github/workflows/helm-base-versions.yaml

Reply:
> It is just a bad practice, we should avoid. this can hadle everything for you https://github.com/hmcts/azure-cftapps-monitoring/blob/master/.github/workflows/helm-base-versions.yaml

Could i get a review in this [PR](https://github.com/hmcts/version-reporter-services/pull/34) as well @adusumillipraveen where i have changed that way `VRS` images are tagged.

Thanks for the pointer 💪🏽